multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1631,6 +1632,7 @@ def _default_predict(
                 model_postprocess_fn=self._model_postprocess_fn,
                 efficient_finetune=OmegaConf.select(self._config, "optimization.efficient_finetune"),
                 trainable_param_names=trainable_param_names,
+                skip_final_val=self._config.optimization.skip_final_val,
```

Comment:
The validation is not called during inference. Maybe no need to set this flag.

Reply:
Good point! Removed it.